src/stability_sdk/animation.py

```diff
@@ -401,6 +427,8 @@ def curve_to_series(curve: str) -> List[float]:
             depth_blur_series = curve_to_series(args.depth_blur_curve),
             depth_warp_series = curve_to_series(args.depth_warp_curve),
             video_mix_in_series = curve_to_series(args.video_mix_in_curve),
+            mask_min_value = curve_to_series(args.mask_min_value),
+            mask_max_value = curve_to_series(args.mask_max_value), 
```

Comment:
I think that the presence itself of `mask_max_value` param is debatable since it is not used directly, but rather strength value are being passed to set upper threshold instead. 
But for the `mask_min_value` I think it should be a time series since it controls the level of 'arbitrariness' that is allowed for inpainting previously occluded zones. For example, when I test rotations around an object, I set `mask_min_value=0.45` because each frame almost the same area is revealed and the inpainting would be too arbitrary and incoherent with lower values. But when I test 'explorations', I encourage emergence of new surprising details that were hidden behind scene objects, so I use `mask_min_value=0.1`.

But I think, your comment is reasonable in a sense that we shouldn't overcomplicate control, so I've removed `mask_max_value` param